drivers/source/SFDP.cpp

```diff
@@ -396,7 +401,17 @@ int sfdp_detect_addressability(uint8_t *bptbl_ptr, sfdp_bptbl_info &bptbl_info)
 
     return 0;
 }
+#elif DEVICE_OSPI
```

Comment:
No need to duplicate code. Please see line 385.

Reply:
If DEVICE_QSPI is not defined in a target, but DEVICE_OSPI is defined, will there be no compilation error?
Thanks!

Reply:
Do you mean like this?
#if (DEVICE_QSPI || DEVICE_OSPI)

Reply:
> 
> 
> Do you mean like this?
> #if (DEVICE_QSPI || DEVICE_OSPI)

Yes, exactly like that.